src/commonTest/kotlin/org/snakeyaml/engine/test_suite/SuiteUtils.kt

```diff
@@ -0,0 +1,114 @@
+package org.snakeyaml.engine.test_suite
+
+import org.snakeyaml.engine.v2.api.LoadSettings
+import org.snakeyaml.engine.v2.api.lowlevel.Parse
+import org.snakeyaml.engine.v2.exceptions.YamlEngineException
+
+internal object SuiteUtils {
+
+    fun parseData(data: YamlTestData): ParseResult {
+        val settings = LoadSettings.builder().setLabel(data.label).build()
+
+        return try {
+            ParseResult(Parse(settings).parseString(data.inYaml).toList())
+        } catch (e: YamlEngineException) {
+            ParseResult(emptyList(), e)
+        }
+    }
+
+    /**
+     * Identifiers of tests that should fail.
+     */
+    val deviationsWithSuccess: Set<YamlTestData.Id> = setOf(
+        //region copied from SnakeYAML
```

Comment:
If snakeyaml-engine knows about these test cases, doesn't it already somehow run this very same test suite?

Reply:
Ok, you answered my question :)